packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -243,26 +228,29 @@ export class PKPBase<T = PKPBaseDefaultParams> {
       throw new Error('pkpPubKey (aka. uncompressPubKey) is required');
     }
 
-    if (this.controllerAuthSig && this.controllerSessionSigs) {
-      throw new Error(
-        'controllerAuthSig, controllerSessionSigs are defined, can only use one or the other'
-      );
-    }
+    this.validateAuthContext();
 
-    // If session sigs are provided, they must be an object
+    // If auth context is provided, it must have a client, props and auth methods
     if (
-      this.controllerSessionSigs &&
-      typeof this.controllerSessionSigs !== 'object'
+      this.authContext &&
+      typeof this.authContext?.client !== 'object' &&
```

Comment:
`typeof` null will also be an object so we gotta check `this.authContext.client !== null` too I think 

<img width="100" alt="image" src="https://github.com/LIT-Protocol/js-sdk/assets/4049673/5c16312d-1a1c-4eab-a575-84733056d034">

Reply:
Changed